packages/clerk-js/src/ui/UserButton/UserButton.tsx

```diff
@@ -10,29 +10,31 @@ import { UserButtonPopover } from './UserButtonPopover';
 import { UserButtonTrigger } from './UserButtonTrigger';
 
 const _UserButton = () => {
+  const { isOpen: isOpenProp } = useUserButtonContext();
   const { floating, reference, styles, toggle, isOpen } = usePopover({
     defaultOpen: false,
     placement: 'bottom-end',
     offset: 8,
+    isOpenProp,
   });
 
   return (
     <Flow.Root flow='userButton'>
       <Flex
         elementDescriptor={descriptors.userButtonBox}
-        isOpen={isOpen}
+        isOpen={isOpenProp || isOpen}
```

Comment:
Hey @raptisj , if a user passes `isOpen={true}`, then the UserButton will stay open and won't respond to user input even after clicking it again, since `isOpenProp || isOpen` will actually be `true || isOpen` that, of course, always evaluates to true. Is that correct?

Reply:
@nikosdouvlis yes that is true. But what is the expected behavior if someone wants to control this as such `isOpen={true}`?